src/components/blog/post/body.module.scss

```diff
@@ -0,0 +1,46 @@
+.root {
+  font-family: "Acta", serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 28px;
+
+  @media (--desktop) {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  @import './body/anchor';
```

Comment:
Probably we could put the imports all at the top of the rule, no?

Reply:
Sure, changed in 93dfdaf.